### PR TITLE
Add an email template for distributors-announce notifications

### PR DESCRIPTION
**Pull request by tallclair**
To help streamline our process, I'd like to add a template for embargoed notifications to distributors-announce. The content is a bit different from the public announcement, but I tried to keep the format close.

/assign @joelsmith @cji

### CODE REVIEW

File: comms-templates/distributors-announcement-email.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+TO: `distributors-announce@kubernetes.io`
+
+SUBJECT: `[EMBARGOED] $CVE: $SUMMARY`
+
+---
+
+### EMBARGOED
+
+The information contained in this email is **[under
+embargo](https://github.com/kubernetes/security/blob/master/private-distributors-list.md#embargo-policy)**
```
Comment (cji) [inline comms-templates/distributors-announcement-email.md:12]: Should we change the format of this to not be a markdown link to make copy/paste of the template easier?

Comment (tallclair) [reply comms-templates/distributors-announcement-email.md:12]: I usually copy the rendered markdown. We should probably decide whether we want to optimize the template for plain-text though. The precedent is to send formatted emails, both for distributors & public announcements, and I haven't heard any complaints.

Comment (cji) [reply comms-templates/distributors-announcement-email.md:12]: That makes sense - copy/pasting the rendered markdown and sending formatted emails sounds good to me. Thank you!

**Pull request merged**